src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/help-galleryImageSpecialized.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    If this field is enabled then osProfile shall be removed from the template as it should not be supplied for Specialized image.
+    When you capture the image from VM, you have an option to do it in a generalized / specialized way. 
```

Comment:
This seems a bit specific to implementation rather than the feature.
how about:
```suggestion
    If you image is <a href="https://docs.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries#generalized-and-specialized-images">Specialized</a> then you need to enable this.
```